src/ui/starter-select-ui-handler.ts

```diff
@@ -989,16 +1017,20 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       return;
     }
 
+    const passiveUnlocked = this.isPassiveUnlocked(species.speciesId);
     const passiveAvailable = this.isPassiveAvailable(species.speciesId);
-    // 'Only Passive Unlocks' mode
+    const valueReductionMaxed = this.isValueReductionMaxed(species.speciesId);
+    const valueReductionAvailable = this.isValueReductionAvailable(species.speciesId);
+    const eggAvailable = this.isSameSpeciesEggAvailable(species.speciesId);
```

Comment:
```suggestion
    const isEggAvailable = this.isSameSpeciesEggAvailable(species.speciesId);
```
`is` (or `has`) prefix helps to quickly distinguish a boolean